zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -225,26 +225,13 @@ fn setup_resolve_common_scalar_after_dispute<T: Config>(
 
 benchmarks! {
     admin_destroy_disputed_market{
-        // a = total accounts
-        // An higher number increases the benchmark runtime significantly, while increasing the
-        // error due to a lower number of repetitions (the data points that are used to approximate
-        // the weight function weight(a) are less precise).
-        // The required weight per account is a linear function of degree 1, i.e.
-        // fn(a) = ba^1 + ca^0. The first few data points of the curve are already approximating
-        // the function and its constant gradient fairly well, provided that these few data points
-        // are very precise (many repetitions). If the gradient is well estimated, any value can be
-        // derived, up to infinity, assuming that at no point the function fn(a) is non-linear.
-        let a in 0..10;
-        // b = num. accounts with assets
-        // Unfortunately frame-benchmarking does not allow to b = b.min(a) here
-        let b in 0..10;
-        // c = num. asset types
-        let c in (T::MinCategories::get().into())..T::MaxCategories::get().into();
-        // Complexity: c*a + c*b ∈ O(a)
-        let c_u16 = c.saturated_into();
-        let (caller, market_id) = setup_resolve_common_categorical::<T>(a, b, c_u16)?;
-
-        for i in 0..c.min(T::MaxDisputes::get()) {
+        let total_accounts = 1000;
+        let accounts_with_assets = 750;
+        let categories = T::MaxCategories::get();
+        let (caller, market_id) = setup_resolve_common_categorical::<T>(total_accounts, accounts_with_assets, categories)?;
+
+        let categories : u32 = categories.saturated_into();
+        for i in 0..categories.min(T::MaxDisputes::get()) {
             let origin = caller.clone();
             let disputes = crate::Disputes::<T>::get(market_id);
             let market = T::MarketCommons::market(&Default::default()).unwrap();
```

Comment:
This will be taken care of in https://github.com/zeitgeistpm/zeitgeist/pull/812 , correct?

Reply:
@Chralt98 ?

Reply:
Correct 👍